PlaidLink.tsx

```diff
@@ -28,7 +28,12 @@ export default function PlaidLink({
   const handleNavigationStateChange = (event: any) => {
     if (event.url.startsWith('plaidlink://')) {
       console.log(event.url)
-      const eventParams = queryString.parse(event.url)
+      const eventParams = queryString.parse(
+        event.url.replace(
+          /plaidlink:\/\/event|plaidlink:\/\/exit|plaidlink:\/\/connected/gi,
```

Comment:
While this works, is this going to be a bit fragile? Like if they add something other than `event`, `exit`, or `connected` it will break. Can we just search for everything after and including the `?`?

Reply:
Good point. I updated to just replace everything up to and including the `?`. I confirmed this works with queryString.